### PR TITLE
Multiprovider example

### PR DESCRIPTION
**Pull request by chinedu117**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,11 @@ It is expected that SQLAlchemy objects, Panda's [DataFrames](https://pandas.pyda
...
 
 - Create a virtual environment for OpenCDMS development
+- For Linux users, `pyopencdms` requires that you install libmysqlclient-dev. (`sudo  apt install libmysqlclient-dev`)
 - Install dependencies used by `pyopencdms`
 - Clone a copy of the `opencdms-test-data` repository
 
+
 ### Example python commands
 
 > NOTE: The example below is old and will be updated by the end of 2021.
```
Comment (faysal-ishtiaq) [inline README.md:43]: please change

```
connection = os.path.join(
     Path.home(), 'opencdms-dev', 'git', 'opencdms-test-data')
```

to 

```
connection = os.path.join(Path.home(), 'opencdms-dev', 'git', 'opencdms-test-data', 'opencdms_test_data', 'data')
```

else it won't work

File: README.md [hunk 2]
```diff
@@ -105,6 +107,29 @@ Currently, `opencdms` package has 5 providers:
 You can manipulate `opencdms.models.mch.english` models using `mch` provider.
 Here are some examples:
 
+
+First set the required environment variables to point to a running instance of mch english database. Below are the default values used in the configuration: 
+
```
Comment (faysal-ishtiaq) [inline README.md:112]: please put the env vars in backticks. now they are appearing as a single line

**Review by faysal-ishtiaq — CHANGES_REQUESTED** [review]
Everything looks file... just some minor fixes

File: README.md (hunk 1 above)
Comment (chinedu117) [reply README.md:43]: Thank you

**Pull request merged**